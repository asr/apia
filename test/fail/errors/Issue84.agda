{-# OPTIONS --exact-split              #-}
{-# OPTIONS --no-sized-types           #-}
{-# OPTIONS --no-universe-polymorphism #-}
{-# OPTIONS --without-K                #-}

postulate
  D   : Set
  _≡_ : D → D → Set
  a b : D

postulate p : a ≡ b
{-# ATP axiom p #-}

postulate foo : a ≡ b
{-# ATP prove foo #-}
