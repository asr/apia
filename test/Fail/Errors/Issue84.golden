ret > ExitFailure 1
err > apia: at least you need to specify one ATP
err >
